HelpAssistHer/help-assist-her
# Update local db name

## Description
For some reason, I incorrectly named the local db `hah-dev`. This is confusing since that's also the name of the dev database. This PR simply updates the name of the local db.
 
## Test Plan
In order to use the new db name, you should be able to just follow these steps
1. Shutdown the mongo process and the server (in the terminal) and disconnect from the db in robomongo. I'm not 100% sure you need to do all these things, but I did just to be safe.
1. Run `yarn data` to import data into the new db.
1. Connect to the db in robomongo and drop the `hah-dev` db.
1. Go to `http://localhost:4000/verification` and make sure you can get and update a resource successfully.
1. Update the name of the connection in robomongo if needed
